frontend/src/employee-frontend/components/ChildInformation.tsx

```diff
@@ -146,7 +147,7 @@ const layouts: Layouts<typeof components> = {
     { component: 'placements', open: false },
     { component: 'backup-care', open: false },
     { component: 'daily-service-times', open: false },
-    { component: 'vasuAndLeops' as keyof typeof components, open: false },
+    { component: 'childDocuments' as keyof typeof components, open: false },
```

Comment:
Is `as keyof typeof components` needed at all? Other entries don't have it.

In any case, `as const` would be the most type safe option. `as` lets you state false things as facts and is a source of bugs!